### ISSUE TITLE
op/pack.t failures in blead

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#11200]
Migrated from [rt.perl.org#86534](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=86534) (status was 'resolved')

Searchable as RT86534$

**Comment by p5pRT** [comment]
### From @iabyn

#### Created by @iabyn



Just perlbugging the issue Karl reported on p5p with message\-id
&nbsp;    \<4D7E38C7\.7080700@&#8203;khwilliamson\.com>

\(so that I've got a ticket to charge my time against :\-\)

To quote the original email&#8203;:

&nbsp;    From&#8203;: Karl Williamson \<public@&#8203;khwilliamson\.com>
&nbsp;    To&#8203;: Perl5 Porters \<perl5\-porters@&#8203;perl\.org>
&nbsp;    Subject&#8203;: op/pack\.t failures in blead
&nbsp;    Message\-ID&#8203;: \<4D7E38C7\.7080700@&#8203;khwilliamson\.com>

&nbsp;    Earlier this year\, 6 tests started failing in blead in pack\.t on my
&nbsp;    machine with this configuration&#8203;:
&nbsp;    This is perl 5\, version 13\, subversion 10 \(v5\.13\.10\-440\-g72d70e5\)
&nbsp;    built for i686\-linux\-thread\-multi\-64int\-ld

&nbsp;    with DEBUGGING and no optimization\.  No failures were encountered on
&nbsp;    another workspace without DEBUGGING and \-O2\.

&nbsp;    I mentioned this on \#p5p\, and never saw smoke failures for it\.  Then
&nbsp;    it went away\, and I forgot about it\, but it just came back again\.  I
&nbsp;    don't know if its the same 6 failures or not\, but here they are&#8203;:
&nbsp;    op/pack\.t    \(Wstat&#8203;: 0 Tests&#8203;: 14699 Failed&#8203;: 6\)
&nbsp;      Failed tests&#8203;:  3933\, 3938\, 3948\, 3988\, 3993\, 3998


&nbsp;    This time I did a git bisect and came up with this&#8203;:
&nbsp;     commit 92e82afa16f5f1aa1b3e163f6d4656d14c44a4d2
&nbsp;     Author&#8203;: Yves Orton \<demerphq@&#8203;gmail\.com>
&nbsp;     Date&#8203;:   Sat Mar 12 17&#8203;:21&#8203;:54 2011 \+0100

&nbsp;         Fix RT \#84294 /\(\(\\w\+\)\(?\{print $2\}\)\)\{2\,2\}/ problem

&nbsp;    I'm still not seeing this in smokes\.

<details><summary>Perl Info</summary>

```

Flags:
    category=core
    severity=medium

Site configuration information for perl 5.13.10:

Configured by davem at Sat Mar 19 19:51:40 GMT 2011.

Summary of my perl5 (revision 5 version 13 subversion 10) configuration:
  Derived from: 75da9d4c616bae3e6791af93d2ced52dc8080f06
  Platform:
    osname=linux, osvers=2.6.34.8-68.fc13.x86_64, archname=x86_64-linux-thread-multi-ld
    uname='linux pigeon 2.6.34.8-68.fc13.x86_64 #1 smp thu feb 17 15:03:58 utc 2011 x86_64 x86_64 x86_64 gnulinux '
    config_args='-des -Dprefix=/home/davem/perl5/git/bleed-davem.out -Dusedevel -Doptimize=-ggdb3 -Aoptimize=-ggdb3 -Aoptimize=-O0 -Dman1dir=none -Dman3dir=none -DDEBUGGING -Dusemorebits -Dusethreads'
    hint=recommended, useposix=true, d_sigaction=define
    useithreads=define, usemultiplicity=define
    useperlio=define, d_sfio=undef, uselargefiles=define, usesocks=undef
    use64bitint=define, use64bitall=define, uselongdouble=define
    usemymalloc=n, bincompat5005=undef
  Compiler:
    cc='cc', ccflags ='-D_REENTRANT -D_GNU_SOURCE -DDEBUGGING -fno-strict-aliasing -pipe -fstack-protector -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64',
    optimize='-ggdb3 -ggdb3 -O0',
    cppflags='-D_REENTRANT -D_GNU_SOURCE -DDEBUGGING -fno-strict-aliasing -pipe -fstack-protector -I/usr/local/include'
    ccversion='', gccversion='4.4.5 20101112 (Red Hat 4.4.5-2)', gccosandvers=''
    intsize=4, longsize=8, ptrsize=8, doublesize=8, byteorder=12345678
    d_longlong=define, longlongsize=8, d_longdbl=define, longdblsize=16
    ivtype='long', ivsize=8, nvtype='long double', nvsize=16, Off_t='off_t', lseeksize=8
    alignbytes=16, prototype=define
  Linker and Libraries:
    ld='cc', ldflags =' -fstack-protector -L/usr/local/lib'
    libpth=/usr/local/lib /lib /usr/lib /lib64 /usr/lib64 /usr/local/lib64
    libs=-lnsl -lgdbm -ldb -ldl -lm -lcrypt -lutil -lpthread -lc
    perllibs=-lnsl -ldl -lm -lcrypt -lutil -lpthread -lc
    libc=/lib/libc-2.12.2.so, so=so, useshrplib=false, libperl=libperl.a
    gnulibc_version='2.12.2'
  Dynamic Linking:
    dlsrc=dl_dlopen.xs, dlext=so, d_dlsymun=undef, ccdlflags='-Wl,-E'
    cccdlflags='-fPIC', lddlflags='-shared -ggdb3 -ggdb3 -O0 -L/usr/local/lib -fstack-protector'

Locally applied patches:
    


@INC for perl 5.13.10:
    /home/davem/perl5/git/bleed-davem.out/lib/site_perl/5.13.10/x86_64-linux-thread-multi-ld
    /home/davem/perl5/git/bleed-davem.out/lib/site_perl/5.13.10
    /home/davem/perl5/git/bleed-davem.out/lib/5.13.10/x86_64-linux-thread-multi-ld
    /home/davem/perl5/git/bleed-davem.out/lib/5.13.10
    .


Environment for perl 5.13.10:
    HOME=/home/davem
    LANG=en_US.utf8
    LANGUAGE (unset)
    LD_LIBRARY_PATH (unset)
    LOGDIR (unset)
    PATH=/usr/lib64/qt-3.3/bin:/usr/kerberos/sbin:/usr/kerberos/bin:/usr/lib64/ccache:/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/sbin:/sbin:/home/davem/bin
    PERL_BADLANG (unset)
    SHELL=/bin/bash

```
</details>

**Comment by p5pRT** [comment]
### From @iabyn

fixed by cd07c537987a76c934f141ca5bc0309e5a6b2609

**Comment by p5pRT** [comment]
@iabyn - Status changed from 'new' to 'resolved'
